src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/add-lets-encrypt-txt-record.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: How to add a TXT record to configure Let's Encrypt certificate
+description: After you create a Let’s Encrypt certificate in a platform other than Azion, but want to use it with a zone created in Intelligent DNS, you must also create TXT record to add the information.
+meta_tags: secure, intelligent dns, record, certificate, domain, let's encrypt
+namespace: docs_guides_secure_intelligent_dns_lets_encrypt_record
+permalink: /documentation/products/guides/secure/lets-encrypt-record/
+menu_namespace: secureMenu
+---
+
+import Button from '~/components/Button.astro'
+
+:::caution[warning]
+This guide covers a scenario in which you have a Let's Encrypt™ certificate created in a platform *outside* Azion and want to associate it with an Intelligent DNS zone. If you've created a Let's Encrypt certificate directly on Azion, the process is automatic and you don't need to create a TXT record. Find out more in the [Let's Encrypt guide](/en/documentation/products/guides/how-to-generate-a-lets-encrypt-certificate/).
```

Comment:
Sugiro simplificar aqui:

de:  `This guide covers a scenario in which you have a Let's Encrypt™ certificate created in a platform *outside* Azion and want to associate it with an Intelligent DNS zone. If you've created a Let's Encrypt certificate directly on Azion, the process is automatic and you don't need to create a TXT record. Find out more in the [Let's Encrypt guide](/en/documentation/products/guides/how-to-generate-a-lets-encrypt-certificate/).`

para:  (mudar o alerta de `:::caution[warning]` para nível INFO... imagino que seja `:::important[information]`)

`If you've created a Let's Encrypt certificate directly on Azion, the TXT record is automatically handled. Find out more in the [Let's Encrypt guide](/en/documentation/products/guides/how-to-generate-a-lets-encrypt-certificate/).`

MOTIVO: Eu posso querer usar o certificado que estou gerando FORA DA AZION. O fato de precisar criar a entrada TXT não me obriga a, depois disso, importar o certificado resultante no RTM.
Quanto ao caso de uso, a descrição (linha 3) já especifica isso.

Ex.: Estou gerando um certificado para uma aplicação interna que não quero configurar AINDA na Azion.

Reply:
@arturdj posso mudar de warning para `note`, que é menos "chamativo".

a linha 3 não aparece para o usuário quando ele está na página. as descriptions aparecem quando o link é compartilhado ou via pesquisa google e outros, então o caso de uso está aparecendo pela primeira vez.
sobre a obrigação, no texto fala "`want` to associate it", então já não esclareceria?

Reply:
@hannah-sk : sobre mudar para NOTE, OK.
Sobre a linha 3 ser description, OK.
Sobre o texto que fala do `want` [linha 13] , eu entendo que poderia ser:

...
This guide covers a scenario in which you want to configure the DNS TXT record required for a Let's Encrypt™ certificate through DNS-01 Challenge validation.
If you've chosen Azion's Domain feature to create a Let's Encrypt certificate directly from Real-Time Manager for a zone that is fully configured in Intelligent DNS, the entire process is automated, including the handling of the TXT record. Find out more in the [Let's Encrypt guide](/en/documentation/products/guides/how-to-generate-a-lets-encrypt-certificate/).
...